wp-content/themes/workingnyc/views/employer-programs/archive.vue

```diff
@@ -88,22 +88,30 @@
   
   <div v-if="init" v-show="!filtersExpanded">
     <div class="flex mx-auto justify-center gap-x-8 my-5 tablet:my-6 desktop:my-7">
-      <section class="hidden desktop:flex w-[350px] p-3 rounded border border-scale-3">
```

Comment:
This shouldn't have been removed, I'm adding it back

Reply:
There's some extra space is coming at the right side of the labels that's why I removed it.

Reply:
I added `w-full` to the form element below to fix it